test/ClangImporter/Inputs/enum-anon.h

```diff
@@ -3,15 +3,23 @@ enum {
   Constant2
 };
 
-enum {
+enum
+#if defined(_WIN32)
+: unsigned
+#endif
```

Comment:
I still don't like this because we need to test the behavior of inferring the type.

Reply:
Hmm, suggestions on how to handle that then?  Unfortunately, it means that it is imported with differing types, and `-verify` can't really deal with that.

Reply:
I think "change these to FileCheck" might be the best we get. Alternately, "separate that part of the test and make Windows and non-Windows versions".

Reply:
Alrighty, I'll split that out and do that so that we can get most of the other issues resolved first.